### PR TITLE
[DEV-796] Improvements to pages.siren.io

### PR DESCRIPTION
**Pull request by makstakeda**
BASED ON https://github.com/sirensolutions/sirensolutions.github.io/pull/4

Additionally added the styles:
* updated header styles with hires logo
* moved to flex layout
* added inline block to links to properly hover it when content block is on hover

https://user-images.githubusercontent.com/36245750/175066720-54cb2a23-16ad-46ff-934f-2943fc8b9bfa.mp4

### CODE REVIEW

File: _layouts/default.html [hunk 1]
```diff
@@ -1,8 +1,14 @@
...
+  <script src="https://kit.fontawesome.com/fdeff5f0cb.js" crossorigin="anonymous"></script>
+</head>
+
 <header class="page-header"
         role="banner">
+
   <link rel="stylesheet" type="text/css" href="/assets/main.css">
   <div id="header-logo">
-		<a href="https://siren.io"><img src="/assets/images/Siren-small_V1.1.png"></a>
+		<a href="https://siren.io"><img src="/assets/images/siren-logo.png"></a>
```
Comment (colmose) [inline _layouts/default.html:10]:
```suggestion
      <a href="https://siren.io"><img src="/assets/images/siren-logo.png"></a>
```

**Review by colmose — APPROVED** [review]
Just a few picky comments, looks good otherwise

**Comment by colmose** [comment]
@andrewgdotcom we will need an entry for `addons.siren.io` instead of `pages.siren.io`

File: assets/main.scss [hunk 2]
```diff
@@ -14,36 +14,100 @@ body {
...
   flex-direction: column;
-  height: 100vh;
-}
+  padding: 0px;
+
+  [class^="fa-"], [class*=" fa-"] {
+    display: inline-block;
+    vertical-align: top;
+  }
+  .icon-big {font-size:55px;color:#f4af1a;}
```
[the remark this answers is not in the document]
Comment (makstakeda) [reply assets/main.scss:26]:
```suggestion
  .icon-big {
    font-size: 55px;
    color: #f4af1a;
  }
```

**Comment by andrewgdotcom** [comment]
> @andrewgdotcom we will need an entry for `addons.siren.io` instead of `pages.siren.io`

Same IP address?

**Comment by makstakeda** [comment]
@andrewgdotcom should be the same. we have to change dns record in here only. i think o___O

**Pull request merged**